src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/RijndaelManaged.cs

```diff
@@ -0,0 +1,110 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.ComponentModel;
+
+namespace System.Security.Cryptography
+{
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public sealed class RijndaelManaged : Rijndael
```

Comment:
They are the same. We just wanted to return an opaque\internal implementation type from .Create() like the other SymmetricAlgorithms, instead of returning the public RijndaelManaged. RijndaelManaged should have never been public :(